### PR TITLE
Fixed sessions widget JS error for novnc connections

### PR DESCRIPTION
**Pull request by abujeda**
The active sessions widget throws a JS error when the active session has a novnc connection info.
The error in the console is:` tryUpdateSetting not defined`

This is because the session panel connection info for novnc connections calls:
```
      tryUpdateSetting(name);
      installSettingHandlers(name);
```
Which are defined in the `batch_connect_sessions.js` which was not included when the active sessions widget is used.

### CODE REVIEW

**Review by johrstrom — DISMISSED** [review]

File: apps/dashboard/app/views/widgets/_sessions.html.erb [hunk 1]
```diff
@@ -6,7 +6,9 @@
 -%>
 
 <%- unless session_selection.empty? %>
-  <%= javascript_include_tag 'sessions_poller', nonce: true %>
```
Comment (johrstrom) [inline apps/dashboard/app/views/widgets/_sessions.html.erb:9]: Do we not need `sessions_poller` at all?

Comment (abujeda) [reply apps/dashboard/app/views/widgets/_sessions.html.erb:9]: I guess we don't as it was the sessions widget the one that was using it.
Will add the `session_poller` code into `batch_connect_sessions.js`

Comment (abujeda) [reply apps/dashboard/app/views/widgets/_sessions.html.erb:9]: This is completed now.

cc: @hmdc/siddev

**Review by johrstrom — APPROVED** [review]
works for me.

**Pull request merged**